crates/hc_bundle/src/packing/mod.rs

```diff
@@ -4,12 +4,20 @@
 
```

Comment:
I'd prefer this file still be named `packing.rs`. It should still be possible to have `packing/wasmer_{sys,wamr}.rs` sub-mods.

Reply:
renamed